lib/rows.js

```diff
@@ -274,6 +272,15 @@ function queryRowsOperationImpl(self, builtPlan, streamType, options, bindingArg
 return operation;
 }
 
+function createRowsEndpoint(options, graphqlQuery) {
+  let endpoint = graphqlQuery?'/v1/rows/graphql':'/v1/rows';
+  if(options && options.update){
+    endpoint = options.structure?'/v1/rows/update?output='+options.structure:
```

Comment:
I think `structure` should be getting applied elsewhere already, since it's an existing feature. Try removing that from here, it shouldn't be needed. It should suffice to just determine which of the 3 endpoints are needed here.

Reply:
Yes structure is used elsewhere as well. Here, it is used to determine the return type of the response (output=array/object). If we remove it we will not get any response. A better approach would be to use a different variable so that in future if "structure" has other values apart from array and object, optic-update would not be affected.

Reply:
Merging this in. Created a new ticket for making /v1/rows/update?output=object as default unless the user overrides it or if execute is used. Making '/v1/rows/update?output=object' results in failure of few remove and transformDoc tests that might need further investigation.